index.html

```diff
@@ -222,78 +223,174 @@ <h2>
-        the constraints of <var>data</var>, the algorithm returns a card as a
-        <a>BasicCardResponse</a>.
-      </p>
-      <ol class="algorithm" data-link-for="BasicCardResponse">
-        <li data-link-for="BasicCardRequest">Let <var>requestedNetworks</var>
-        be the result of filtering <var>data</var>["<a>supportedNetworks</a>"]
-        for <a>networks</a> that are <a>known</a> by this payment handler.
-        </li>
-        <li data-link-for="BasicCardRequest">Let <var>requestedTypes</var> be
-        <var>data</var>["<a>supportedTypes</a>"].
-        </li>
-        <li>Let <var>card</var> be a <a>BasicCardResponse</a>.
-        </li>
-        <li>Set <var>card</var>["<a>cardNumber</a>"] to a string of digits of
-        length between 10 to 19 items representing the <a>primary account
-        number</a>. The <a>primary account number</a> SHOULD be one of the <a>
-          types</a> from <var>requestedTypes</var>, or any <a>type</a> if
-          <var>requestedTypes</var> is empty.
-        </li>
-        <li>Check that <var>card</var>["<a>cardNumber</a>"] is from one of the
-        <a>networks</a> in <var>requestedNetworks</var>, or any <a>network</a>
-        if <var>requestedNetworks</var> is empty. Optionally, validate
-        <var>card</var>'s <a>details</a> to make sure they adhere to any
-        <a>type</a> and/or <a>network</a> requirements.
-          <div class="note" title="Validation of inputs">
+      <section>
+        <h3>
+          Can a payment handler make a payment?
+        </h3>
+        <p>
+          The algorithm to check if a Payment Handler that handles "basic-card"
+          <dfn>can make a payment</dfn> takes <a>BasicCardRequest</a>
+          <var>request</var> as input. It returns either true or false:
+        </p>
+        <ol class="algorithm">
+          <li>Let <var>cards</var> be a <a data-cite="infra#list">list</a> of
+          <a>cards</a> associated with this payment handler.
+          </li>
+          <li>If <var>cards</var> <a data-cite="infra#list-is-empty">is
+          empty</a>, return false.
+          </li>
+          <li>Let <var>networks</var> be an empty <code><a data-cite=
+          "!WEBIDL#idl-DOMString">DOMString</a></code> sequence.
+          </li>
+          <li>Let <var>types</var> be an empty <a>BasicCardType</a> sequence.
+          </li>
+          <li>If <var>request</var>["supportedTypes"] is present, append each
+          item in <var>request</var>["supportedTypes"] to <var>types</var>.
+          </li>
+          <li>If <var>request</var>["supportedNetworks"] is present, append
+          each item in <var>request</var>["supportedNetworks"] to
+          <var>networks</var>.
+          </li>
+          <li>For each <var>card</var> in <var>cards</var>:
+            <ol>
+              <li>If <a>instrument is supported</a> <var>card</var>,
+              <var>networks</var>, <var>types</var> returns true, return true.
```

Comment:
Hi @marcoscaceres, line 258 looks to be missing some punctuation (I think).

Reply:
thanks! Will fix.

Reply:
https://github.com/w3c/payment-method-basic-card/pull/54